### PR TITLE
added my new blog

### PR DESCRIPTION
**Pull request by supunTE**
Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

I added a new blog of mine.
https://stedirisuriya.github.io/gci17.fossasia.org/

### CODE REVIEW

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@STEdirisuriya please use different image for your blog avoid using same image again and again

**Review by anshumanv — CHANGES_REQUESTED** [review]
Update image

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Please add the required image

**Comment by supunTE** [comment]
Changed the photo

**Review by abishekvashok — APPROVED** [review]
LGTM, but will be better of you could squash the commits.

**Comment by NaviRocker** [comment]
Better to use an image which is relevant to your blog

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Agree with @naveentherocker and @abishekvashok  though :smile:

File: _data/blogs.yml [hunk 1]
```diff
@@ -97,3 +97,8 @@
   title: MY EXPERIENCE OF GOOGLE CODE – IN
   img: aks.jpg
   author_name: Ashish Kumar Singh
+
+- url: https://morethanworld64569539.wordpress.com/2017/12/12/how-i-fixed-an-issue-by-20-steps/
+  title: How I fixed an issue? – By 20 steps
+  img: supun3.png
```
Comment (ParthS007) [inline _data/blogs.yml:103]: image
both are same image @STEdirisuriya

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Comment by supunTE** [comment]
You know there is problem with my Windows local installation I said it for you all. Please think about that first. Please say me resolve for that. How I do squash without using terminal? I do this thing using online method of GitHub.

File: _data/blogs.yml (hunk 1 above)
Comment (supunTE) [reply _data/blogs.yml:103]: What is the problem? Please see above of Blogs section there are three same photos. Two picture are from one person. And other same picture from another person. May be twins.

Comment (supunTE) [reply _data/blogs.yml:103]: See it first.

**Comment by NaviRocker** [comment]
Try this
git reset --soft HEAD~No.OfCommits
git commit -m "<Your commit message>"
git push -f origin gh-pages

**Comment by ba11b0y** [comment]
@STEdirisuriya Editing code online on Github is never a good practice and will trouble you in the long run.
Please mention your problem with the installation on the gitter channel and we will help you resolve it and also you will realise the power of git's CLI.

**Comment by STEdirisuriya** [comment]
I said I'm using online Please think it. Do you want to do any other thing please say me a way to local host it.

**Comment by supunTE** [comment]
@invinciblycool I say about my error with lot of mentors and students. No answer yet. What I have to do?
Aren't sure? Please ask from @anshumanv  @abishekvashok @ParthS007

**Comment by ba11b0y** [comment]
@STEdirisuriya Please use gitter for conversation irrelevant to this PR or issue. 😄

**Comment by anshuman73** [comment]
Hi, seems you've tagged the wrong person. Please be careful, I'm probably not who you want to refer to @STEdirisuriya

**Comment by supunTE** [comment]
@anshuman73 Sorry

**Comment by NaviRocker** [comment]
@STEdirisuriya Please Address your Problem properly. So Mentors will surely help you

**Review by NaviRocker — APPROVED** [review]
Well just check out for Grammar mistakes on your blog.

**Pull request closed without merging**